### PR TITLE
Install golang via gimme

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: features/Dockefile.buildimage [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
 
 # rpms required for building and running test suites
 RUN dnf -y install \
-    $GOVERSION \
+    gcc \
+    git \
     make \
-    tar \
     gettext \
+    which \
```
Comment (davidvossel) [inline features/Dockefile.buildimage]: i think the pkg changes here don't apply.

Comment (cynepco3hahue) [reply features/Dockefile.buildimage]: what do you mean by pkg changes?

Comment (davidvossel) [reply features/Dockefile.buildimage]: the addition/removal of the install packages. those changes don't seem relevant here.

**Comment by davidvossel** [comment]
/retest

```
Last metadata expiration check: 0:00:01 ago on Wed Jan 29 15:24:51 2020.
No match for argument: golang-1.13.5-1.fc31
Package tar-2:1.32-2.fc31.x86_64 is already installed.
```

weird

**Comment by slintes** [comment]
> /retest
> 
> ```
> Last metadata expiration check: 0:00:01 ago on Wed Jan 29 15:24:51 2020.
> No match for argument: golang-1.13.5-1.fc31
> Package tar-2:1.32-2.fc31.x86_64 is already installed.
> ```
> 
> weird

CI always takes the Dockerfile from master... so we just need to merge this, and then see on another PR if it works. Very annoying, but we went through this in the 2 orher repos already...

/lgtm

File: features/Dockefile.buildimage (hunk 1 above)
Comment (MarSik) [reply features/Dockefile.buildimage]: Hmm.. I can see how git, which, findutils and python2 are relevant. Even gcc. But does gimme really require skopeo?

Comment (slintes) [reply features/Dockefile.buildimage]: good catch, no (copy paste from other repo where it's needed indeed)

Comment (cynepco3hahue) [reply features/Dockefile.buildimage]: fixed

**Comment by slintes** [comment]
I'm going to merge this in order to move forward :)

**Comment by slintes** [comment]
works, `unit-tests` are back to green :)

**Pull request merged**